towncrier/newsfragments/1341.removal

```diff
@@ -0,0 +1 @@
+Remove startup resolution of HOST_* variables to *_ADDRESS
```

Comment:
Normally the file name points to the PR (#1716), not to the initial issue. That way you can click on it in releases.rst and jump directly to the PR that introduced the described change.